### DISCUSSION TITLE
Buisness plan

### DISCUSSION THREAD

**Discussion by Olyno** [deta/cloud-docs#382] [General] [answered]
Hi 👋🏻 

I discovered your service recently, and I was wondering what your business plan was. You offer a lot of things, but I hardly see how you can be profitable.

Do you have a plan for that?

**Comment by ghost** [comment, marked as answer]
Detas new product will be money generating

**Reply by Olyno** [reply]
I don't think that new products will pay for all the infrastructure needed to deploy and use the current service. Everything (except storage which is limited) is free and unlimited, I doubt that creating a new product will pay for that.

**Reply by abdelhai** [reply]
if you doubt that, what is the solution in your opinion?

**Reply by Olyno** [reply]
Putting limits on freebies is a possibility. You can give very interesting offers to developers, without giving everything for free. If developers want more, they pay.

You can also make a "pay-as-you-go" offer, which allows developers to lower the price of their bills, and will give you the money to pay for the infrastructure and teams behind it, allowing for growth.

**Reply by abdelhai** [reply]
thanks for sharing your thoughts! this is definitely a viable strategy yet it is not completely fair to all developers. especially if they live in a country where credit cards and paypal are not supported or if they are under the age of 18/16. deta wants to give everyone the same access to the same tools and platforms to everyone, no matter where they live, etc...

we've been working on deta for many years and developed our own strategy which will start to unfold this fall.

**Reply by Olyno** [reply]
Another idea that could be interesting in these cases is to accept crypto-currencies as a means of payment. This is one of the principles of crypto-currencies, the fact that anyone with an internet connection can own crypto-currency.

This is of course just ideas, but I think it could set deta apart from the competition, as not many hosting companies accept crypto currency as a means of payment.

You could even create a "connect with" and let developers connect with a wallet in 2 clicks.

**Reply by edge-rps** [reply]
I think in the long run the unlimited free model has much more potential financially than a limited free model, as it incentivizes making quality apps without worrying about funding. I could be wrong but maybe it's worth a shot.

**Comment by ghost** [comment]
I guess we will have subscriptions soon

**Reply by abdelhai** [reply]
subscriptions are not planned
